### ISSUE TITLE
Custom fee value changes to negative integer when user inputs low custom value

### ISSUE THREAD

**Issue by Eshwari007** [blockstack/blockstack-app#2268]
The user was testing as part of regression and found the bug which always seems to sneak in occasionally but is difficult to be reproduced.
Attaching the screen recording of a low custom fee changes to a negative value for perusal
Also noticed when purchasing NFT via any 3 rd party sites, the wallet seems to be opening twice with double popup wallet loading and the fee confirmation had to be entered twice as well. This seems to be an intermittent behavior as well.

CC ing @landitus for design input as well on the behavior mentioned above

https://user-images.githubusercontent.com/38872890/155588348-47fa58d4-5aa3-4b6c-8f0c-3ac8bb589d1d.mov

**Comment by markmhendrickson** [comment]
Are two separate popup windows opening and staying open here at the beginning?

<img width="326" alt="image" src="https://user-images.githubusercontent.com/28991/155680207-fd7d71bf-b367-46e3-8f92-08102ccf755b.png">

**Comment by fbwoolf** [comment]
The double popup typically happens to me when I accidentally have a prod and dev version of the wallet running at the same time, might this be the case here?

**Comment by Eshwari007** [comment]
Good catch @fbwoolf , Yes I had two wallet versions for comparison hence noticed the double popup.
Tested with just one wallet and seems to have the fix the popup issue.

**Comment by fbwoolf** [comment]
Do you think that might have also caused the negative fee? Just curious bc we don't seem to see it reported from users?

**Comment by Eshwari007** [comment]
No @fbwoolf ,this was one of the bugs which I was able to reproduce several times in the past but wasn't able to record it as it was very intermittent. It seems to occur usually the first time after I download a new extension and enter the custom fee quickly with the same exact amount `"0.000772"` or lower than that. This was my first time having two wallet versions opened up simultaneously hence am pretty sure I only had one active wallet in the past. I will monitor and keep an eye on the bug as well.
